CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## In Development
+* Fix st2web ingress which should have been defined as an Integer instead of a String.
```

Comment:
```suggestion
* Fix st2web ingress which should have been defined as an Integer instead of a String (#111)
```

Now that we have a PR #, would be nice to reference it in the Changelog